File: 136-single-number/136-single-number.go
func singleNumber(nums []int) int {
    res_dict := make(map[int]int)
    for i:=0;i<len(nums);i++{
        res_dict[nums[i]] = res_dict[nums[i]] + 1 
    }
    
    for k,v := range res_dict{
        if v == 1{
            return k
        }
    }
    
    return -1
}

File: 136-single-number/136-single-number.py
class Solution:
    def singleNumber(self, nums: List[int]):
        res_dict = {}
        
        for i in nums:
            res_dict[i] = res_dict.get(i, 0) + 1
        
        for k, v in res_dict.items():
            if v == 1:
                return k
            
        